src/components/CreateMessage.tsx

```diff
@@ -0,0 +1,78 @@
+import { Button, Form, Input, InputNumber } from "antd";
+import { addMessage } from "../services/chain/apis/extrinsic";
+import { SchemaProps } from "../services/types";
+import { staticSchema } from "./RegisterSchema";
+
+
+
+const CreateMessage = (props: SchemaProps): JSX.Element => {
+    const [form] = Form.useForm();
+
+    const example_msg = { nickname:'omar',favorite_number: 6,favorite_restaurant:'Ramen Takeya'};
+
+    const handleSubmit =  async () => {
+
+        let isValid = validateMessage();
+        if (isValid) {
```

Comment:
I'd likely collapse this.

```suggestion
        if (validateMessage()) {
```